cube/src/main/java/org/jboss/arquillian/ce/cube/CEEnvironmentProcessor.java

```diff
@@ -88,22 +88,60 @@
      * objects, e.g. CreateCube(application), StartCube(application)
      * <p>
      * Needs to fire before the containers are started.
+     * @throws Exception 
      */
     public void createEnvironment(@Observes(precedence = 10) BeforeClass event, OpenShiftAdapter client,
-                                  CECubeConfiguration configuration, OpenShiftClient openshiftClient) throws DeploymentException {
+                                  CECubeConfiguration configuration, OpenShiftClient openshiftClient) throws Exception {
```

Comment:
It seems that throwing a generic Exception is not a good practice in Java. Is there a strong reason for that?